fe/fe-core/src/main/java/com/starrocks/catalog/Catalog.java

```diff
@@ -7096,6 +7096,41 @@ public void replayDropFunction(FunctionSearchDesc functionSearchDesc) {
         db.replayDropFunction(functionSearchDesc);
     }
 
+    public void setHasForbitGlobalDict(String dbName, String tableName) {
+        Map<String, String> emptyProperty = new HashMap<>();
+        Database db = getDb(dbName);
+        if (db == null) {
+            throw new Error("the DB " + dbName + "isn't  exist");
+        }
+
+        Table table = db.getTable(tableName);
+        if (table == null) {
+            throw new Error("the DB " + dbName +  " tabet: " + tableName + "isn't  exist"); 
+        }
+
+        OlapTable olapTable = (OlapTable) table;
+        olapTable.setHasForbitGlobalDict();
+        ModifyTablePropertyOperationLog info = new ModifyTablePropertyOperationLog(db.getId(), table.getId(), emptyProperty);
```

Comment:
Should we put the the bool value to the properties?  If in future, we want to allow the forbidden table global dict optimization.